.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Shell scripts
+*.sh text eol=lf
```

Comment:
To clarify, what OS were your builds breaking on?

Reply:
My OS is windows, and the container was Linux. From what I understand, I pulled it to my OS and the line endings where changed. When I tried to run it failed because it was the line endings used in windows instead of the line endings needed for Linux.

Reply:
Thanks for the info. Makes sense since need to keep the linux line endings since they are being copied into a linux docker.

Reply:
Would expand on your comment, so we don't forget what it is for:
```
# Shell scripts need to maintain linux line endings even in windows environments (since they are being copied into linux docker environments).
```